packages/@uppy/transloadit/src/Assembly.js

```diff
@@ -1,18 +1,16 @@
-const Emitter = require('component-emitter')
-const has = require('@uppy/utils/lib/hasProperty')
-const NetworkError = require('@uppy/utils/lib/NetworkError')
-const fetchWithNetworkError = require('@uppy/utils/lib/fetchWithNetworkError')
-const parseUrl = require('./parseUrl')
-
-// Lazy load socket.io to avoid a console error
+import Emitter from 'component-emitter'
+import socketIo from 'socket.io-client'
+import has from '@uppy/utils/lib/hasProperty'
+import NetworkError from '@uppy/utils/lib/NetworkError'
+import fetchWithNetworkError from '@uppy/utils/lib/fetchWithNetworkError'
+import parseUrl from './parseUrl.js'
+
+// We used to lazy load socket.io to avoid a console error
 // in IE 10 when the Transloadit plugin is not used.
 // (The console.error call comes from `buffer`. I
 // think we actually don't use that part of socket.io
 // at all…)
-let socketIo
 function requireSocketIo () {
```

Comment:
Might as well remove this function if it directly returns socketIO?

Reply:
Ah yes, I forgot about this change. This could be a breaking change based on the comment, we should probably keep the `require` call for the time being.

Reply:
Addressed in https://github.com/transloadit/uppy/commit/c2d1693f4a90e839ed91441dc3c7787f99917582, PTAL